core/arch/riscv/plat-spike/platform_config.h

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#define TEE_RAM_START		CFG_TZDRAM_START
```

Comment:
TZ in `CFG_TZDRAM_START` means trustzone. I'd prefer a more accurate name.

Reply:
I would use Trusted Domain, does `CFG_TDDRAM_START` work ?

Reply:
Sounds good.